lib/4_filter_numbers.js

```diff
@@ -6,6 +6,8 @@
 
 module.exports = {
   filter_numbers: function(arr){
-    // Your code goes here.
+    var newarr = arr.filter(function(x){ return x < 10; })
+               .map(function(x) { return x});
```

Comment:
Honestly I got a little help on this one from an example on an MDN page, but their example didn't have a .map in it. The function within the filter specifies filtering any value less than 10 and returns the results as an array, so it was redundant to include the .map iteration. In fact, I have no idea why I did that or where I got it from. It works without it, so I removed it.